### ISSUE TITLE
Colour contrast in "Go to Your Profile" Button in reg'n confirmation email is insufficient

### ISSUE THREAD

**Issue by gityakk** [davidpettersson/ridehub#97]
David, I find the contrast between the text blue against the background blue is insufficient and makes the text very hard to read.
